content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -61,15 +61,17 @@ and `SERVICE_ACCOUNT`
 
     {{< /tabset >}}
 
-1. Create the working directory:
+1. Create the working directory on your cluster:
 
     {{< text syntax=bash snip_id=setup_wd >}}
     $ mkdir -p "${WORK_DIR}"
     {{< /text >}}
 
 ## Install the Istio control plane
 
-Install Istio and expose the control plane so that your virtual machine can access it.
+If your cluster already installed the Istio control plane, you can skip this step.
```

Comment:
Still need to expose the control plane if you already have it installed.

Reply:
Ok :)